exercises/practice/roman-numerals/tests/test-roman-numerals.art

```diff
@@ -0,0 +1,139 @@
+import {unitt}!
+import {src/roman-numerals}!
+
+suite "Roman Numerals" [
+    test "1 is I" [
+        result: as.roman 1
```

Comment:
I thought that using the `as` word here would be cool. What do you think? I have no idea if my example implementation makes any sense.

Reply:
I'll follow up on the Arturo Discord server later.

Reply:
Try to use `not? null?`. 
I'll discuss this with Yanis about this behavior, since `ensure` is only analyzing `:logical` types.

Reply:
To avoid any kind of strange behavior, please, use:

```art
ensure -> true? attr 'roman
```

This will avoid your user to pass other types rather than a `:logical` for instance: `as.roman: "Empire"` should not be accepted, as well `as.roman: false` should not be accepted too.

But, `as.roman` or `as.roman: true` should. 🙂

Reply:
`attr?` won't pop your attributes from the stack, so you can have inconsistent behavior between callings.

`attr` pops your attribute, but I would recomend to check its type first. My default recommendation:

```art
; Ensure if the type is right, and duplicate the value into a variable using `<=`/`dup`:
ensure -> block? examples: <= attr 'examples
; Use it as you want
functionUsing examples
```

Reply:
We probably want to use a different function name as well so we don't imply that the student is actually extending the `as` built-in with an additional attribute. 

`as`, `convert`, `render`, `to`, `translate` are used already so maybe `stringify` would work here. In this hypothetical, `stringify` returns the number as a string, but `stringify.roman` returns it as the corresponding Roman numerals. That could be extended to include other output formats like `stringify.binary`.

If we set up tests in between that call `stringify` without the `roman` attribute, then we can make sure students are properly taking handling the stack. This exercise is coded as medium difficulty so I think this complication is fair game.

Reply:
@BNAndras Do we have enough to go on now?

Reply:
I think so. We have two potential routes I see and I don't have a strong preference. Either we forgo the attribute and use `toRoman` as a regular function or we use `stringify.roman`, adding a test at the end where `stringify 1` returns `"1"` and not `"I"`. That'll make the students deal with the stack before finishing the exercise without adding too much difficulty up front.